cpp/models/ode_seir/model.h

```diff
@@ -63,6 +64,52 @@ class Model : public CompartmentalModel<InfectionState, Populations<InfectionSta
         dydt[(size_t)InfectionState::Recovered] =
             (1.0 / params.get<TimeInfected>()) * y[(size_t)InfectionState::Infected];
     }
```

Comment:
The whole file is not correctly formatted. Please use clang-format before making pull request and on every commit.